keps/693-multikueue/README.md

```diff
@@ -316,6 +328,81 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### MultiKueue Dispatcher API
+
+Since Kueue 0.13, in order to meet the requirements of [Story 3](#story-3), we introduce an API for custom dispatching algorithms.
+When a custom Dispatcher API is used, instead of creating the copy of the Workload on all clusters the
+the MultiKueue Workload Controller only creates the copy of the Workload on the subset of worker clusters
+specified in the Workload's .spec.nominatedClusterNames field.
+
+Additionally, we implement a built-in incremental dispatcher as a reference implementation. 
+Including the pre-existing dispatching algorithm we distinguish the following dispatchers:
+
+* **AllAtOnce**:  
+The workload is copied to all available worker clusters at once. This is the default dispatching algorithm.
+
+* **Incremental**:  
+Clusters are nominated incrementally in rounds of fixed duration (5 minutes per round). 
+The process begins by nominating an initial set of 3 clusters, which are set in the `.spec.nominatedClusterNames` field in the Workload.
+If none of the clusters admit the workload within the current round's duration, the next round begins, 
+and 3 additional clusters are nominated, until the workload is admitted or all eligible clusters have been considered.
+This strategy allows for a controlled and gradual expansion of candidate clusters, rather than dispatching the workload to all clusters at once.
+
+* **External**:  
+The selection of worker clusters is delegated to an external controller. 
+The external controller is responsible for setting the `.spec.nominatedClusterNames` field to the names of the selected clusters.
+If the nominated clusters field is changed by the external controller, workloads are removed from any clusters that are no longer nominated.
+While the workload is in the admitted state, the `MultiKueueNominatedClusters` field is immutable. 
+
+#### Workload Synchronization
+
+While the Workload is pending, the dispatcher can add or remove clusters from the list, and the Workload
+Controller synchronizes the value of the field with the subset of worker clusters with the Workload copy.
+
+Changes to Workload type:
+```go
+type WorkloadSpec struct {
+  // nominatedClusterNames specifies the list of cluster names that have been nominated for scheduling.
+  // This field is optional.
+  // 
+  // +listType=set
+  // +kubebuilder:validation:MaxItems=10
+  // +kubebuilder:validation:XValidation:rule="!(self.status.clusterName != null) || oldObject.spec.nominatedClusterNames == self.spec.nominatedClusterNames",message="nominatedClusterNames is immutable when the workload is assigned to a cluster"
```

Comment:
Does `oldObject.spec.nominatedClusterNames == self.spec.nominatedClusterNames"` validation means that retrying the same clusters is not allowed?

Reply:
Ah, I was misunderstood. You're right.
Thank you.